float64[6] lambda2
float64 lambda
float64 detM
bool power
float64[6] u_conn_x
float64[6] u_conn_y
float64[6] u_ca_x
float64[6] u_ca_y
float64[6] u_fc_x
float64[6] u_fc_y
